bundles/org.eclipse.ui.workbench/Eclipse UI/org/eclipse/ui/IWorkbenchPreferenceConstants.java

```diff
@@ -705,4 +705,13 @@ public interface IWorkbenchPreferenceConstants {
 	 * @since 3.130
 	 */
 	String LARGE_VIEW_LIMIT = "largeViewLimit"; //$NON-NLS-1$
+
+	/**
+	 * <p>
+	 * Whether the UI updates on runtime during dpi change
+	 * </p>
+	 *
+	 * @since 3.133
+	 */
+	String UPDATE_ON_RUNTIME_KEY = "UPDATE_ON_RUNTIME_KEY"; //$NON-NLS-1$
```

Comment:
I'd go with `RESCALING_AT_RUNTIME` since it matches the **public** (API) method `Display::setRescalingAtRuntime` and would adapt the names of other methods, parameters, constants, fields and information texts in this PR too in order to keep it consistent.